### PR TITLE
Ensure sorting analyzer in zarr are consolidated

### PR DESCRIPTION
**Pull request by alejoe91**
As in title

### CODE REVIEW

File: src/spikeinterface/core/sortinganalyzer.py [hunk 1]
```diff
@@ -563,11 +566,13 @@ def create_zarr(cls, folder, sorting, recording, sparsity, return_scaled, rec_at
...
         recording_info = zarr_root.create_group("extensions")
 
+        zarr.consolidate_metadata(zarr_root.store)
+
     @classmethod
     def load_from_zarr(cls, folder, recording=None, storage_options=None):
         import zarr
 
-        zarr_root = zarr.open(str(folder), mode="r", storage_options=storage_options)
+        zarr_root = zarr.open_consolidated(str(folder), mode="r", storage_options=storage_options)
```
Comment (samuelgarcia) [inline src/spikeinterface/core/sortinganalyzer.py:575]: we have several zarr.open in this module can you check then all ?

**Comment by alejoe91** [comment]
@samuelgarcia this is fixed. The failing tests wa sa tricky one to find: basically we have to consolidate metadata AFTER run info is updated (and it was not the case)

**Comment by alejoe91** [comment]
As discussed with @samuelgarcia, I also added the `ext.delete()` function :)

**Review by samuelgarcia — APPROVED** [review]

**Comment by samuelgarcia** [comment]
This sounds a bit annoying to do ` zarr.consolidate_metadata` after eevry changes. No ?
mets got with this for now.

**Pull request merged**